plugins/obs-qsv11/CMakeLists.txt

```diff
@@ -105,17 +51,11 @@ if(OS_WINDOWS)
             device_directx9.cpp
             device_directx9.h)
 
-  target_link_libraries(obs-qsv11 PRIVATE OBS::libmfx d3d9 d3d11 dxva2 dxgi dxguid)
+  target_link_libraries(obs-qsv11 PRIVATE vpl::vpl d3d9 d3d11 dxva2 dxgi dxguid)
```

Comment:
Now that windows and linux are both using the vpl dispatcher, please move linking of vpl to the shared the linkages on line 29.

Reply:
This was resolved.